src/Chapter1.hs

```diff
@@ -568,8 +573,11 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
 
+isVowel :: Char -> Bool
+isVowel c
+    | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' || c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U' = True
+    | otherwise = False 
```

Comment:
If you write it with `||`, I guess there is no much benefits of guards. You can:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' || ...
```

We recommended guards, if you would check each case separately:

```haskell
  | c == 'a'  = True
  | c == 'e' = True
  | ...
```